### ISSUE TITLE
Importing an .FBX with GLB enabled will not properly rename any included animation

### ISSUE THREAD

**Issue by h0lmiew** [playcanvas/engine#2817]
### Description

In attempting to migrate our project to use GLB instead of JSON for FBX files, we have a cumbersome problem, in which any imported FBX that contains an animation, will have the generated animation file named Scene.glb.

When you import as JSON, it will correctly name the generated animation file as what the FBX was originally called. See attached screenshot.

![image](https://user-images.githubusercontent.com/5739868/106479964-2c477d80-64ab-11eb-83d9-5a1955374ef0.png)

Having to manually rename each imported animation is a daunting task when you have a lot of them.

### Steps to Reproduce

1. Drag in FBX file that has an animation in it, with Convert to GLB enabled.
2. Generated model .glb file will be named correctly as the .fbx file
3. Generated animation .glb file will NOT be named correctly as the .fbx file, but will instead always be named Scene.glb.

**Comment by slimbuck** [comment]
Hi @h0lmiew,

The JSON converter only imports the first animation take from the FBX file and so can name it after the FBX file.

The glTF converter however imports all the animation takes found in the file and so uses the take names found in the FBX instead. Going forward it would probably be best to name your takes descriptively so these get used during import.

So sorry for the inconvenience that this change has caused. We actually discussed the naming at length as a team when we added the glTF converter and decided using take names was the lesser of the evils.

Thanks,
Donovan

**Comment by willeastcott** [comment]
Could we consider an Asset Tasks setting which would select a scheme for animation asset naming (maybe `Use Take Name` or something).

**Comment by slimbuck** [comment]
Yeah we could add that. Would we export the first animation only or number the files or something else?

**Comment by mvaligursky** [comment]
It should probably apply to only import of fbx with a single take in it. If there are multiple takes, we'd need to name them somehow.

**Comment by yaustar** [comment]
+1 for @mvaligursky's suggestion

**Comment by willeastcott** [comment]
What do you think @slimbuck?

**Comment by yaustar** [comment]
Spoke to the team about this. The animation name is taken from the take name in the source file of the FBX. It's also worth noting that FBX -> GLB import can handle multiple animations (one FBX file that has many animations produces many GLBs, one for each animation that are named by the name given in the original source).
